content/legal/release-policy.md

```diff
@@ -71,19 +71,19 @@ contain compiled code.
 
 #### Release signing  {#release-signing}
 
-All supplied packages MUST be cryptographically signed with a detached signature.
-It MUST be signed by either the Release Manager or the automated release
+All supplied packages MUST be cryptographically signed with an ASCII-armored detached signature.
+They MUST be signed by either the Release Manager or the automated release
 infrastructure, where the underlying implementation MUST follow the principles
 [outlined](/dev/release-signing.html#automated-release-signing) by the Apache
-Security Team. All supplied packages MUST use a detached signature. Those who
+Security Team. Those who
 vote +1 for release MAY offer their own cryptographic signature to be concatenated
 with the detached signature file (at the Release Manager's discretion)
 prior to release.
 
 #### Compiled packages  {#compiled-packages}
 
 The Apache Software Foundation produces open source software. All releases
-are in the form of the source materials needed to make changes to the
+are in the form of the source materials needed to build the
```

Comment:
I think the 'make changes to' might be relevant here, referring to the definition of "Source" in the Apache License ("the preferred form for making modifications, including but not limited to software source code, documentation source, and configuration files."). So maybe "modify and build"?